pkg/prommetrics/prommetrics.go

```diff
@@ -109,6 +109,16 @@ var (
 		},
 		[]string{"type", "namespace"},
 	)
+
+	scaledObjectPauseTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: DefaultPromMetricsNamespace,
+			Subsystem: "scaled_object",
+			Name:      "pause",
```

Comment:
Is this the metric name? If so, we should align it with the rest of the names to be more consistent

Reply:
At the end the name should look like this - keda_scaled_object_pause
Do you have any other suggestions?

Reply:
LGTM